lib/services/role.go

```diff
@@ -443,22 +430,20 @@ func (r *RoleV3) SetRules(rct RoleConditionType, rrs map[string][]string) {
 	}
 }
 
-// GetSystemResources gets all low level resources.
-func (r *RoleV3) GetSystemResources(rct RoleConditionType) map[string][]string {
-	if rct == Allow {
-		return r.Spec.Allow.SystemResources
+// applyImplicitRules sets implicit rules if they don't exisit in rule map so
+// we always have the necessary minimum set of rules.
+func applyImplicitRules(rules map[string][]string) {
```

Comment:
applyImplicitRules will add the rules to the database what we agreed to avoid. We agreed to add a role with implicit rules to the role set.